utils/order-utils/test/orders.js

```diff
@@ -28,7 +28,7 @@ describe('Orders', async () => {
 
   const NOV_7_2020_22_18_14 = '1604787494'
   const MAY_11_2017_00_00_00 = '1494460800'
-
+  const INVALID_KIND = '0xFFFFFF'
```

Comment:
What do you mean by 'invalid'? Just that it hasnt been registered yet?

Reply:
meaning that it is not ERC20 or ERC721, this doesn't use anything related to the transfer-registry, prior the we used for the word 'INVALID_KIND' however toBuffer requires a hex string, so I just moved it out to be a const variable.